README.md

```diff
@@ -28,6 +28,10 @@ mkdir -p "$HOME/bin" &&
 curl -L https://github.com/qlik-oss/corectl/releases/download/v0.0.2/corectl-Windows-x86_64.exe > "$HOME/bin/corectl.exe" &&
 chmod +x "$HOME/bin/corectl.exe"
 ```
+Optionally you can add the bin to the PATH environment variable to simplify usage in the command line.
+```bash
+setx PATH "%PATH%;%HOME%/bin"
```

Comment:
I'm a bit hesitant to tips like this unless it is something needed specifically for this tool. The problem is often that the commands are OS specific and might require elevated privileges.

Reply:
Agree. I am leaning to more cluttering of the docs.